model/src/main/kotlin/CopyrightFinding.kt

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (C) 2017-2019 HERE Europe B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package com.here.ort.model
+
+import com.fasterxml.jackson.core.JsonParser
+import com.fasterxml.jackson.databind.DeserializationContext
+import com.fasterxml.jackson.databind.JsonNode
+import com.fasterxml.jackson.databind.deser.std.StdDeserializer
+import com.fasterxml.jackson.module.kotlin.treeToValue
+
+import com.here.ort.utils.SortedSetComparator
+import com.here.ort.utils.textValueOrEmpty
+
+import java.util.SortedSet
+import java.util.TreeSet
+
+data class CopyrightFinding(
+        val statement: String,
+        val locations: SortedSet<TextLocation>
+) : Comparable<CopyrightFinding> {
+    companion object {
+        private val LOCATIONS_COMPARATOR = SortedSetComparator<TextLocation>()
```

Comment:
How about moving this as `SORTED_SET_COMPARATOR` to `TextLocation` (next to its existing `COMPARATOR`) and using that constant then from both `CopyrightFinding` and `LicenseFinding` classes?

If doing that, consequently `val COPYRIGHTS_COMPARATOR = SortedSetComparator<CopyrightFinding>()` should also be moved from `LicenseFinding` to `CopyrightFinding` as `SORTED_SET_COMPARATOR`.

Reply:
Done for both.